my-app/.gitignore

```diff
@@ -5,9 +5,6 @@
 /.pnp
 .pnp.js
 
-# testing
-/coverage
-
```

Comment:
Do not remove this folder from .gitignore. No need to push coverage info to the remote repository as it can be generated locally by running the coverage command.